stackstorm-compose/st2.user.conf

```diff
@@ -0,0 +1,6 @@
+# /etc/st2/st2.user.conf
+#
+# This is override st2 config file which will be passed to any st2 command, like this:
```

Comment:
Same as on the previous comment above.

Reply:
This file was originally copied from base/files/tmp/st2.user.conf, but I have rewritten the comment for stackstorm-compose.